packages/dds/tree/src/domains/json/jsonDomainSchema.ts

```diff
@@ -29,64 +29,40 @@ const jsonTypes: Set<TreeSchemaIdentifier> = new Set();
 
 const json: NamedTreeSchema[] = [];
 
-export const jsonObject: NamedTreeSchema = {
+export const jsonObject: NamedTreeSchema = namedTreeSchema({
```

Comment:
nit: We probably don't need the type annotation anymore.

Reply:
I considered removing them. Since this schema kind of serves as an example, I left the overly explicit types think they might make it slightly easier to understand and navigate to the relevant types. Could really go either way though.

Reply:
Sounds good, I don't have a strong opinion either way.